src/Epic/Dog.js

```diff
@@ -0,0 +1,19 @@
+import { combineEpics, ofType } from 'redux-observable'
+import * as Dog from '../Redux/State/Dog'
+import { mergeMap, map } from 'rxjs/operators'
+import { invoker } from 'ramda'
+
+export const fetchDogEpic = (action$, state$, { fetchDog }) => action$.pipe(
+  ofType(Dog.GENERATE_DOG),
+  mergeMap(fetchDog),
+  mergeMap(invoker(0, 'json')),
```

Comment:
why a mergeMap here ?
```javascript
map(response => response.json())
```
should be enough IMHO

Reply:
AFAIR The fetch api returns a promise of a response so you have to merge'em all before using the response